circus/tests/test_client.py

```diff
@@ -78,41 +77,51 @@ def test_client(self):
         yield self.stop_arbiter()
 
 
+TMP_FILE = './HOOK_CALLED'
+
+
 def long_hook(*args, **kw):
-    time.sleep(5)
+    with open(TMP_FILE, 'a'):
```

Comment:
Strange. For me it fails at the last assertion (assertFalse for the existence of file)